woocommerce/woocommerce-android
# Onboarding: Get paid task

Fixes #8532.

This PR adds the Get Paid onboarding task implementation. It adds handlers for both `payments` and `woocommerce-payments` task types (they're mutually exclusive).

If the site is self-hosted, the WebView is not authenticated and the user needs to enter user credential to proceed with the payments setup. For WP.com hosted sites the authentication is automatic.

A different payments setup URL is loaded if:

- A site doesn't have the WooCommerce Payments plugin: `<url>/wp-admin/admin.php?page=wc-settings&tab=checkout`
- A site has the WooCommerce Payments plugin: `{site_url}/wp-admin/admin.php?page=wc-admin&task=payments`

**To test:**

Test 4 different combinations of sites and verify they behave as expected:

1. Self-hosted site without WC Payments plugin: Get paid task opens the login screen & after authentication loads `<url>/wp-admin/admin.php?page=wc-settings&tab=checkout`
2. Self-hosted site with WCP Payments plugin: Get paid task opens the login screen & after authentication loads `{site_url}/wp-admin/admin.php?page=wc-admin&task=payments`
3. Atomic site without WC Payments plugin: The same URL is loaded as self-hosted but without the need to authenticate
4. Atomic site with WC Payments plugin: The same URL is loaded as self-hosted but without the need to authenticate
